src/components/Popover/Popover.js

```diff
@@ -54,7 +54,10 @@ class Popover extends Component {
 
   handleClickOutside = (e) => {
     if (this.state.open && this.wrapper && !this.wrapper.contains(e.target)) {
-      this.setState({ open: false });
+      this.setState({ open: false }, () => {
+        this.props.onClose(this.state, e);
```

Comment:
do we've a convention about what to pass to callback? i'm passing this (though not using it for my current use case)